fe/.eslintrc.js

```diff
@@ -26,10 +33,40 @@ module.exports = {
     // Ignore `import` statements with Webpack `@` alias in path
     'import/no-unresolved': ['error', { ignore: ['^@'] }],
 
+    // Set maximum line length to 120 characters. Lines containing URLs, strings, template literals or regular
+    // expressions will be ignored.
+    'max-len': ['error', {
+      'code': 120,
+      'ignoreUrls': true,
+      'ignoreStrings': true,
+      'ignoreTemplateLiterals': true,
+      'ignoreRegExpLiterals': true
+    }],
+
     // Do not permit `console` references or `debugger` statements in production bundle
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
+    // Requires the use of single quotes wherever possible. Double quotes are allowed as long as the string contains
```

Comment:
I guess **use** -> **usage** because of **the** article is used only with noun or groups of nouns.
```suggestion
    /* Requires the usage of single quotes wherever possible. Double quotes are allowed as long as the string contains a single quote that would have to be escaped otherwise. 
    */
```

Reply:
Agreed. Let's replace "use" with "usage". But should we use `/* ... */` for this comment only? Is there any problem with `//`?

Reply:
I guess **no**. Just pointing out to the accessed feature.

Reply:
There is no difference between [use](https://translate.google.com/#view=home&op=translate&sl=en&tl=ru&text=use) and [usage](https://translate.google.com/#view=home&op=translate&sl=en&tl=ru&text=usage).

Reply:
No. See difference please [use](https://en.oxforddictionaries.com/definition/use) vs. [usage](https://en.oxforddictionaries.com/definition/usage)

Reply:
@dmitriyace please resolve this conversation